profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -199,13 +219,114 @@ fun UserTabs(modifier: Modifier = Modifier) {
+    MANGA("Manga"),
+    FAVOURITES("Fave"),
+    STATISTICS("Stats")
+}
+
+@Composable
+fun AboutTab(
+    viewer: Viewer,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.padding(
+            top = LocalPaddings.current.large,
+            start = LocalPaddings.current.large,
+            end = LocalPaddings.current.large
+        ),
+        verticalArrangement = Arrangement.spacedBy(LocalPaddings.current.medium)
+    ) {
+        Genres(viewer.genres)
+    }
+}
+
+@Composable
+fun Genres(
+    genres: List<Viewer.Genre>,
+    modifier: Modifier = Modifier
+) {
+    val barWidthAnimation = remember { Animatable(0f) }
+    val barAlphaAnimation = remember { Animatable(0f) }
+    val barColor = MaterialTheme.colorScheme.primary
+    LaunchedEffect("barAnimation") {
+        launch {
+            barWidthAnimation.animateTo(
+                targetValue = 1f,
+                animationSpec = tween(durationMillis = 1000, delayMillis = 250)
+            )
+        }
+        launch {
+            barAlphaAnimation.animateTo(
+                targetValue = 1f,
+                animationSpec = tween(durationMillis = 1000, delayMillis = 250)
             )
         }
     }
+
+    Text(
+        text = "Genres",
+        style = MaterialTheme.typography.titleMedium
+    )
+    Row(modifier.height(IntrinsicSize.Max)) {
+        Column(horizontalAlignment = Alignment.End) {
+            genres.forEach {
+                Text(
+                    text = it.genre,
+                    style = m3TextStyles().textStyle.copy(
+                        color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.74f)
+                    )
+                )
+            }
+        }
+        Column(
+            modifier = Modifier
+                .fillMaxHeight()
+                .padding(
+                    start = LocalPaddings.current.small,
+                    end = LocalPaddings.current.large
+                ),
+            verticalArrangement = Arrangement.spacedBy(LocalPaddings.current.tiny)
+        ) {
+            val highestCount = genres.maxOf { it.mediaCount }
+            genres.forEach {
+                val weight = it.mediaCount/highestCount.toFloat()
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth(fraction = weight)
+                        .weight(1f)
+                        .graphicsLayer { alpha = weight * barAlphaAnimation.value }
+                        .drawBehind {
+                            drawRoundRect(
+                                color = barColor,
+                                size = Size(
+                                    width = size.width * barWidthAnimation.value,
+                                    height = size.height
+                                ),
+                                cornerRadius = CornerRadius(size.height)
+                            )
+                        },
+                ) {  }
```

Comment:
Is this supposed to be empty? We should have a Box overload that doesn't need a content

Reply:
Yeah, it's just the bar in the bar graph